File: new_restaurantelar_admin/src/app/models/create_pedido_dto.ts
export class CreatePedidoDto {
    
    fullNameClient: string;
    typePayment: number;
    idsPlato: number[];
    dateReserva: string;
    numMesa: number;
    type: number;
    idWaiter: number;
    
    constructor() {
      this.fullNameClient = '';
      this.typePayment = 0;
      this.idsPlato = [];
      this.dateReserva = '';
      this.numMesa = 0;
      this.type = 0;
      this.idWaiter = 0 ;
    }
  }

File: New Restaurantelar API/src/main/java/com/salesianos/triana/proyecto/Restaurantelar/dto/CreatePedidoDto.java
package com.salesianos.triana.proyecto.Restaurantelar.dto;

import com.salesianos.triana.proyecto.Restaurantelar.model.Plato;
import lombok.*;

import java.time.LocalDateTime;
import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class CreatePedidoDto {

    private String fullNameClient;
    private int type;
    private Long idWaiter;
    private List<Long> idsPlato;
    private int typePayment;
    private LocalDateTime dateReservation;
    private int numMesa;

}
